Source/WebCore/platform/graphics/cg/GraphicsContextCG.cpp

```diff
@@ -1073,6 +1073,23 @@ static void setCGShadow(CGContextRef context, RenderingMode renderingMode, const
         CGContextSetShadowWithColor(context, CGSizeMake(xOffset, yOffset), blurRadius, cachedCGColor(color).get());
 }
 
+#if PLATFORM(COCOA)
```

Comment:
I generally prefer for these to be a bit more narrowly scoped when we can, and I think we can in this case. Maybe HAVE(CORE_GRAPHICS_CREATE_SHADOW2) or something to that effect?

Reply:
Done. I used `HAVE(CORE_GRAPHICS_FILTERS)` to gird this code. If more CoreGraphics styles are added in the future, new HAVE macros will be added.